cmd/account/list/account.go

```diff
@@ -111,11 +114,18 @@ func (o *listAccountOptions) run() error {
 		accounts        awsv1alpha1.AccountList
 		outputAccounts  awsv1alpha1.AccountList
 		resourcePrinter printers.ResourcePrinter
+		accountId       string
 		matched         bool
 		reused          bool
 		claimed         bool
 		err             error
 	)
+
+	if o.accountID != "" {
+		inputListTag := &organizations.ListTagsForResourceInput{
+			ResourceId: aws.String(o.accountID)}
+		_, err = aws
```

Comment:
Could you elaborate on this line? as far as I can see it's not used anywhere and the error is not checked.

Reply:
Oh i shouldn't have committed that. It's from another ticket that i started working on